nucypher/policy/policies.py

```diff
@@ -96,15 +86,33 @@ class TreasureMapPublisher:
     log = Logger('TreasureMapPublisher')
 
     def __init__(self,
-                 worker,
+                 treasure_map_bytes,
                  nodes,
+                 network_middleware,
                  percent_to_complete_before_release=5,
                  threadpool_size=120,
                  timeout=20):
 
         self._total = len(nodes)
         self._block_until_this_many_are_complete = math.ceil(len(nodes) * percent_to_complete_before_release / 100)
-        self._worker_pool = WorkerPool(worker=worker,
+
+        def put_treasure_map_on_node(node):
```

Comment:
Thank you for that, I should've done it when I made this class. There was no reason to keep the worker function outside.

Reply:
👍